api/src/main/java/com/capitalone/dashboard/service/BuildServiceImpl.java

```diff
@@ -162,7 +168,9 @@ public BuildDataCreateResponse createV3(BuildDataCreateRequest request) throws H
             throw new HygieiaException(e);
         }
         finally {
-            populateDashboardId(response);
+            if(BooleanUtils.isTrue(settings.isLookupDashboardForBuildDataCreate())) {
```

Comment:
@rvema  remove BooleanUtils and use implicit condition checking 
if(settings.isLookupDashboardForBuildDataCreate()))

Reply:
Booleanutils is null safe hence it was used , any specific  reason behind removing it?

Reply:
addressed it.